server/aesdsocket.c

```diff
@@ -267,10 +271,23 @@ void *packet_handler(void *conn_param)
         #if !USE_AESD_CHAR_DEVICE
         pthread_mutex_lock(&lock);
         #endif
-        write_to_file(f_fd, read_buf, len);   
-        printf("\n%s %ld\n",read_buf,conn_data->thread_id);
-        send_packets(f_fd, conn_data->accepted_fd);
 
+        if(strncmp(read_buf, ioctl_command, strlen(ioctl_command)) == 0)
+        {
+          printf("\nInside IOCTL\n");
+          struct aesd_seekto seekto;
+          sscanf(read_buf, "AESDCHAR_IOCSEEKTO:%d,%d", &seekto.write_cmd, &seekto.write_cmd_offset);
```

Comment:
@kevintom98 you need error checking on `sscanf` here to handle cases where user input is malformed.